CogServer/app.py

```diff
@@ -53,17 +74,40 @@ def tile(
     cm = cmap.get(color)
     options={"unscale":unscale}
     with Reader(dataset, options=options) as cog:
-        # print(cog.info())
         img = cog.tile(x, y, z)
     img.rescale(
-        in_range=((0, max_val),),
+        in_range=((min_val, max_val),),
         out_range=((0, 255),)
     )
     content = img.render(img_format="PNG", colormap=cm, **img_profiles.get("png"))
     # print(img.data)
     # return 200
     return Response(content, media_type="image/png")
 
+
+@app.get("/get_rgb_tile/{z}/{x}/{y}.png")
+def get_rgb_tile(
+    z:int, 
+    x:int, 
+    y:int, 
+    dataset: str,
+    unscale: bool = True
+    ):
+    '''Returns a Mapbox-ready elevation tile'''
+    options = img_profiles.get('png')
+    dataset = f'{GCS_BASE}/{dataset}'
+    options={"unscale":unscale}
+    with Reader(dataset, options=options) as cog:
+        t = cog.tile(x, y, z)
+        buff = render(
+            to_rgb(t.data[0], cog.info().nodata_value),
```

Comment:
Is this pretty fast? Seems like a good opportunity for preprocessing (and saving new files) if there are any performance concerns

Reply:
It's fast <3